Sources/FlexLayout.swift

```diff
@@ -664,7 +664,7 @@ public final class Flex {
       This method is valid only when the item position is absolute (`view.flex.position(.absolute)`)
       */
     @discardableResult
-    public func horizontally(_ percent: FPercent) -> Flex {
+    public func horizontally(_ percent: FPercent = 0%) -> Flex {
```

Comment:
This method change is not required, the one with the `CGFloat` parameter will be enough to support this feature.